### PR TITLE
[Enhancement] (nereids)implement adminCleanTrashCommand in nereids

### LINKED ISSUE #42846: [Enhancement] (nereids)implement adminCleanTrashCommand in nereids
adminCleanTrash command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following adminCleanTrash command in nereids:
unsupportedadminCleanTrashStatement
``` ADMIN CLEAN TRASH (ON LEFT_PAREN backends+=STRING_LITERAL
        (COMMA backends+=STRING_LITERAL) RIGHT_PAREN)?                              ```

### PR DESCRIPTION
**Pull request by DongLiang-0**
### What problem does this PR solve?

Issue Number: close #42846

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCleanTrashCommand.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    }
+
+    public AdminCleanTrashCommand(List<String> backendsQuery) {
+        super(PlanType.CLEAN_TRASH_COMMAND);
+        this.backendsQuery = backendsQuery;
+    }
+
+    @Override
+    public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        handleCleanTrash(backendsQuery);
```
Comment (LiBinfeng-01) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCleanTrashCommand.java:57]: we should check privilege before run clean trash
```
if (!Env.getCurrentEnv().getAccessManager().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
            ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "ADMIN");
        }
        ```

Comment (DongLiang-0) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCleanTrashCommand.java:65]: done

**Comment by DongLiang-0** [comment]
run buildall

**Review by LiBinfeng-01 — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCleanTrashCommand.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * admin clean trash
+ */
+public class AdminCleanTrashCommand extends Command {
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCleanTrashCommand.java:47]:
```suggestion
public class AdminCleanTrashCommand extends Command implements NoForward {
```

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCleanTrashCommand.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            for (String backendQuery : backendsQuery) {
+                if (backendsID.containsKey(backendQuery)) {
+                    needCleanedBackends.add(backendsInfo.get(backendsID.get(backendQuery)));
+                    backendsID.remove(backendQuery);
+                }
+            }
+        }
+        return needCleanedBackends;
+    }
+
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCleanTrashCommand.java:94]: override checkSupportedInCloudMode method and implement as checkDdlStmtSupported() method in DdlExecutor.java like bellow
```
@Override
protected void checkSupportedInCloudMode(ConnectContext ctx) throws DdlException {
    LOG.info("AdminCleanTrashCommand not supported in cloud mode");
    throw new DdlException("Unsupported operation");
}
```

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Review by LiBinfeng-01 — APPROVED** [review]

**Review by starocean999 — DISMISSED** [review]

**Comment by DongLiang-0** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Pull request merged**